rest/data/project_test.go

```diff
@@ -20,6 +24,46 @@ type ProjectConnectorGetSuite struct {
 	suite.Suite
 }
 
+const (
+	projectId      = "mci2"
+	username       = "me"
+	projEventCount = 10
+)
+
+var sampleProjectSettings = model.ProjectSettings{
```

Comment:
I would avoid having package level mutable values in tests, under any circumstance.